bin/tbds*

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
```

Comment:
@dimanyc What does the `*` in this filename represent? I don’t think I’ve seen that before and I’m curious.

Reply:
From @dimanyc - “that's a vim thing showing files with specific permission modifiers. I think we can drop the * here”